_component_design/tables-controls.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Tables - Controls
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table controls refer to elements which may be present on any table variation that allows users to interact with the data contained within a table. These controls are a collection of smaller elements such as search bars, buttons, pagination controls, action cogs, amoung others. 
+preview-image: preview-images/tables-controls.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### When to use
+{: .hxSubSectionTitle}
+The table pattern has multiple controls that are used to interact with the contents displayed within a table. Controls may be placed in three primary locations.:
+
+- [On top of the table]({{page.url}}#top)
+- [Below the table]({{page.url}}#below)
+- [Within the table itself]({{page.url}}#within)
+
+{% endcolumn %}
+
+{% column right:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+{% figure [caption:"Table snippets showing the three locations for table controls. Actual controls present depend on the specific table needs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-controls-hero.svg){:width="100%"}
```

Comment:
The intro is hard to parse.  Consider removing it or chunking it out into separate images/sub sections.

![image](https://user-images.githubusercontent.com/933210/31954998-ab4c5b2c-b8ac-11e7-9565-aa950c1042d9.png)

Reply:
Responded to every other feedback item from Laura and yourself. This one will take a little more cognitive lifting and the world series game is almost on. I'll handle this one prior to the review tomorrow AM. Will respond to this comment once done.

Reply:
This graphic was split into three sections. I trimmed the margins on each SVG to be flush with the svg graphical elements and the SVGs still were not clear in the 4/8 col layout. As a one off this section was converted to a 3/9 layout. 
![image](https://user-images.githubusercontent.com/586682/31997824-fd2cfcca-b952-11e7-8f4a-9959c3dd86b0.png)